validator/testcases/javascript/instanceproperties.py

```diff
@@ -225,6 +252,8 @@ def set_contentScript(value, traverser):
                         "is dangerous and error-prone. Please use a separate "
                         "JavaScript file, along with the "
                         "`contentScriptFile` property instead.",
+            signing_help="Please do not use the `contentScript` property "
+                         "in any add-ons submitted for automated signing.",
```

Comment:
We'll currently accept it in any add-ons submitted for manual code review (listed add-ons), just not add-ons submitted for automated signing. Which means that if those add-ons wind up in the review queue with this issue, they'll be rejected.